.vscode/settings.json

```diff
@@ -1,4 +1,7 @@
 {
     "editor.formatOnSave": true,
-    "rust-analyzer.cargo.features": "all"
-}
+    "rust-analyzer.cargo.features": "all",
+    "vscode-nmake-tools.workspaceBuildDirectories": [
+        "."
+    ]
+}
```

Comment:
```suggestion
}

```

Reply:
That change shouldn't have snuck in, removing.